js/controllers/dialog.js

```diff
@@ -29,16 +29,24 @@ DialogController.prototype.show = function(callback) {
   this.container_.find('.dialog-button').first().focus();
 };
 
+/**
+ * Disables keyboard tabbing to all UI elements outside of the dialog box.
```

Comment:
`@private` for both `disableEverything_` and `reenableEverything_`

Reply:
Done